### PR TITLE
Update status badge

### PR DESCRIPTION
**Pull request by damccorm**
Switch appveyor badge to prettier badge. Reacts to #5410 

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,8 @@
     <img alt="Travis CI Build Status" src="https://img.shields.io/travis/prettier/prettier/master.svg?style=flat-square&label=Travis+CI"></a>
   <a href="https://circleci.com/gh/prettier/prettier">
     <img alt="CircleCI Build Status" src="https://img.shields.io/circleci/project/github/prettier/prettier/master.svg?style=flat-square&label=CircleCI"></a>
-  <a href="https://ci.appveyor.com/project/prettier/prettier">
-    <img alt="AppVeyor Build Status" src="https://img.shields.io/appveyor/ci/prettier/prettier.svg?style=flat-square&label=AppVeyor"></a>
+  <a href="https://dev.azure.com/prettier/prettier/_build/latest?definitionId=5">
+    <img alt="Azure Pipelines Build Status" src="https://dev.azure.com/prettier/prettier/_apis/build/status/prettier.prettier"></a>
```
Comment (ikatyang) [inline README.md]: Is there a flat style badge? something like badge.

Comment (ikatyang) [inline README.md]: Can we show only the status on master?

Comment (damccorm) [reply README.md]: Sorry, missed that this was the standard. Updated appropriately

Comment (damccorm) [reply README.md]: Good catch, updated appropriately.

File: README.md [hunk 2]
```diff
@@ -41,8 +41,8 @@
     <img alt="Travis CI Build Status" src="https://img.shields.io/travis/prettier/prettier/master.svg?style=flat-square&label=Travis+CI"></a>
   <a href="https://circleci.com/gh/prettier/prettier">
     <img alt="CircleCI Build Status" src="https://img.shields.io/circleci/project/github/prettier/prettier/master.svg?style=flat-square&label=CircleCI"></a>
-  <a href="https://ci.appveyor.com/project/prettier/prettier">
-    <img alt="AppVeyor Build Status" src="https://img.shields.io/appveyor/ci/prettier/prettier.svg?style=flat-square&label=AppVeyor"></a>
+  <a href="https://dev.azure.com/prettier/prettier/_build/latest?definitionId=5">
+    <img alt="Azure Pipelines Build Status" src="https://img.shields.io/azure-devops/build/prettier/prettier/5.svg?branchName=master&style=flat-square&label=Azure%20Pipelines"></a>
```
Comment (j-f1) [inline README.md]: Seems like the second `prettier` has to be the internal project ID, i.e. `https://img.shields.io/azure-devops/build/prettier/79013671-677c-4846-a6d8-3050d40e21c0/5.svg?style=flat-square&label=Azure+Pipelines`.

**Pull request merged**